src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Figure {
+    private String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    public String infoFromFigure() {
+        return "";
```

Comment:
The `infoFromFigure()` method currently returns an empty string. If this method is intended to provide information about the figure, consider implementing it to return meaningful data.